docs/docs/javascript/overview/enabling-typescript-support.mdx

```diff
@@ -0,0 +1,168 @@
+---
```

Comment:
I've not yet checked if this renders correctly - I couldn't see any obvious guidance on this.

Reply:
I've checked this now thanks to Benjy's docs - rearranged a bit after review. 

Side note - all the javascript doc content is nested under 'overview' - should that be removed at some point (separately)? 

<img width="290" height="208" alt="Image" src="https://github.com/user-attachments/assets/c49f5673-e86b-4637-9952-c783d7fa948d" />